hivemq4/dns-image/pre-entry.sh

```diff
@@ -6,6 +6,4 @@ if [[ "${HIVEMQ_CLUSTER_TRANSPORT_TYPE}" == "UDP" ]]; then
 elif [[ "${HIVEMQ_CLUSTER_TRANSPORT_TYPE}" == "TCP" ]]; then
     # shellcheck disable=SC2016
     sed -i -e 's|<\!--TRANSPORT_TYPE-->|<tcp><bind-address>${HIVEMQ_BIND_ADDRESS}</bind-address><bind-port>${HIVEMQ_CLUSTER_PORT}</bind-port></tcp>|' /opt/hivemq/conf/config.xml
-fi
-
-exec /opt/docker-entrypoint.sh "$@"
\ No newline at end of file
+fi
```

Comment:
Removing the `exec` statement will break "backwards compatibility" - do we then still want to keep `/opt/pre-entry.sh` and symlink it?

Reply:
what do you think? do you think it's a better idea to just have both in that case? or just remove this one so the user can copy it back in if needed?